pytest-selenium/utils/utility.py

```diff
@@ -209,291 +210,10 @@ def page_highlights(cls, driver) -> int:
-            "chapter_search_results_total": 14,
-            "rkt_search_results_total": 1,
-        },
-        "principles-microeconomics-2e": {
-            "default_page": "1-introduction",
-            "search_term": "Explicit costs",
-            "chapter_search_results_total": 18,
-            "rkt_search_results_total": 1,
-        },
-        "principles-macroeconomics-ap-courses-2e": {
-            "default_page": "1-introduction",
-            "search_term": "adjustable-rate mortgage",
-            "chapter_search_results_total": 14,
-            "rkt_search_results_total": 1,
-        },
-        "principles-microeconomics-ap-courses-2e": {
-            "default_page": "1-introduction",
-            "search_term": "positive externality",
-            "chapter_search_results_total": 24,
-            "rkt_search_results_total": 1,
-        },
-        "principles-managerial-accounting": {
-            "default_page": "1-why-it-matters",
-            "search_term": "relevant range",
-            "chapter_search_results_total": 51,
-            "rkt_search_results_total": 3,
-        },
-        "principles-management": {
-            "default_page": "1-introduction",
-            "search_term": "plan is a decision to carry out a particular action",
-            "chapter_search_results_total": 11,
-            "rkt_search_results_total": 0,
-        },
-        "psychology-2e": {
-            "default_page": "1-introduction",
-            "search_term": "event schema",
-            "chapter_search_results_total": 15,
-            "rkt_search_results_total": 1,
-        },
-        "statistics": {
-            "default_page": "1-introduction",
-            "search_term": "memoryless property",
-            "chapter_search_results_total": 5,
-            "rkt_search_results_total": 1,
-        },
-        "university-physics-volume-1": {
-            "default_page": "1-introduction",
-            "search_term": "interference",
-            "chapter_search_results_total": 66,
-            "rkt_search_results_total": 3,
-        },
-        "university-physics-volume-2": {
-            "default_page": "1-introduction",
-            "search_term": "interference fringes",
-            "chapter_search_results_total": 3,
-            "rkt_search_results_total": 0,
-        },
-        "university-physics-volume-3": {
-            "default_page": "1-introduction",
-            "search_term": "interference fringes",
-            "chapter_search_results_total": 61,
-            "rkt_search_results_total": 1,
-        },
-        "us-history": {
-            "default_page": "1-introduction",
-            "search_term": "PATRIOTS",
-            "chapter_search_results_total": 20,
-            "rkt_search_results_total": 0,
-        },
-    }
+    # Read the books details from books.json file
+    book_details = open("/Users/malar/Rex/rex-web/pytest-selenium/utils/books.json", "r")
+    books = json.loads(book_details.read())
```

Comment:
Since you modified the import, this should be `loads(` instead of `json.loads(`; I expect this to throw a `NameError` unless JSON is imported elsewhere before it gets to the utilities file.

Reply:
yes my bad. corrected now @gregfitch